src/test/java/com/google/javascript/clutz/extern_enum.externs.js

```diff
@@ -0,0 +1,4 @@
+var ns = {};
+
+/** @enum */
+ns.Enum = {A: 0}
```

Comment:
The goal was not to emit the value separately too. Previously, we emitted:

declare namespace ಠ_ಠ.clutz.ns.Enum {
  var A: number;
}
along what you see in extern_enum.d.ts